legal-api/src/legal_api/models/filing.py

```diff
@@ -67,6 +67,7 @@ class Status(Enum):
     _status = db.Column('status', db.String(10), default='DRAFT')
     paper_only = db.Column('paper_only', db.Boolean, unique=False, default=False)
 
+    effective_date = db.Column('effective_date', db.DateTime(timezone=True), default=None)
```

Comment:
This is already in line 63, effective_date should not be defined twice.

Reply:
good catch, I had this defined before the filer split went in, and when I was cherry-picking commits one of them must've had this